generators/client/templates/angular/src/main/webapp/app/_app.module.ts

```diff
@@ -30,11 +39,11 @@ import { <%=angularXAppName%>AdminModule } from './admin/admin.module';
 import { <%=angularXAppName%>AccountModule } from './account/account.module';
 <%_ } _%>
 import { <%=angularXAppName%>EntityModule } from './entities/entity.module';
-import { customHttpProvider } from './blocks/interceptor/http.provider';
 import { PaginationConfig } from './blocks/config/uib-pagination.config';
-
+<%_ if (authenticationType === 'session' || authenticationType === 'oauth2') { _%>
```

Comment:
use `['session', 'oauth2'].includes(authenticationType)` syntax as its easier to read